places-database-management-1/bulk-import-of-pois.md

```diff
@@ -0,0 +1,96 @@
+Places Import script
+=================
+A set of Python scripts have been created to simplify the batch import of POIs from a .csv file into a Places database by using the Places REST APIs. These scripts can be downloaded from this open source [git repo](https://github.com/adobe/places-scripts). Please check the [Organizational Requirements](../places-rest-apis/organizational-requirements.md) and [Adobe I/O integation](../places-rest-apis/adobe-i-o-integration/README.md) sections before running  these script to ensure you have access to the Places REST APIs.
```

Comment:
- Please change "Places database" to "POI database"
- Please change "Places REST APIs" to "Web Service APIs"

- Reword the 3rd sentence so that it reads as follows:
"Before you run these scripts, to ensure you have access to the Web Service APIs, see [Organizational Requirements](../places-rest-apis/organizational-requirements.md) and [Adobe I/O integation](../places-rest-apis/adobe-i-o-integration/README.md)."